mail-client/clawsker/clawsker-1.3.5.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils
+
+DESCRIPTION="Applet to edit Claws Mail's hidden preferences"
+HOMEPAGE="https://www.claws-mail.org/clawsker.php"
+SRC_URI="https://www.claws-mail.org/tools/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-lang/perl
+	dev-perl/Gtk3
+	dev-perl/Locale-gettext
+	>=dev-perl/File-Which-1.210
+	mail-client/claws-mail
+"
+DEPEND="test? ( dev-perl/Test-Exception )"
+
+src_install() {
```

Comment:
I wonder, if you drop this, does it install?

Reply:
given juippis' comment, I guess the PREFIX part is needed :)

Reply:
@juppis , @thesamesam , can we set `PREFIX` without `src_install` overload?

Reply:
You can most definitely patch it, but 
```
src_install() {
	emake install DESTDIR="${D}" PREFIX=/usr
}
```
seems simpler in every way. Make sure the docs are being installed, and/or add `einstalldocs` if they're not.

Reply:
If upstream's alive, a patch works a bit better because it means we can drop src_install.